app/routes/overview/components/NextEvent.css

```diff
@@ -12,6 +12,13 @@
   margin: 0;
 }
 
+.info {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin: -3px 10px;
+}
+
 .info span {
   margin: 0 10px;
 }
```

Comment:
Remove this as afaics it's not used anymore

Reply:
It's used by the text to the right of the icon ("Påmelding åpnet...")